python/plugins/processing/algs/help/qgis.yaml

```diff
@@ -0,0 +1,442 @@
+qgis:addautoincrementalfield: >
+  This algorithm adds a new integer field to a vector layer, with a sequential value for each feature.
+
+  This field can be used as a unique ID for features in the layer.
+
+  The new attribute is not added to the input layer but a new layer is generated instead.
+
+qgis:addfieldtoattributestable: >
+  This algorithm adds a new attribute to a vector layer.
+
+  The name and characteristics of the attribute are defined as parameters.
+
+  The new attribute is not added to the input layer but a new layer is generated instead.
+
+qgis:adduniquevalueindexfield: >
+  This algorithm takes a vector layer and an attribute and adds a new numeric field. Values in this field correspond to values in the specified attribute, so features with the same value for the attribute will have the same value in the new numeric field. This creates a numeric equivalent of the specified attribute, which defines the same classes.
+ 
+  The new attribute is not added to the input layer but a new layer is generated instead.
+
+qgis:advancedpythonfieldcalculator: >
+  This algorithm adds a new attribute to a vector layer, with values resulting from applying an expression to each feature. The expression is defined as a Python function.
+
+  More information can be found <a href ="http://gis-lab.info/qa/fieldpyculator.html">here</a>
```

Comment:
Do we want to link non-english (cyrillic) pages from here (?)
If yes, I would at least make people aware.
And use a descriptive text for the link instead of "here".